graph.py

```diff
@@ -0,0 +1,71 @@
+from itertools import product
+import random
+from collections import deque
+
+class Vertex():
+    def __init__(self,name:str,value=None,children:list=None):
+        if name is not None:
+            self.name = name
+        if value is not None:
+            self.value = value
+        else:
+            self.value = None
+
+        if children is not None:
+            self.children = children
+        else:
+            self.children = []
+            # maybe cache misses will be bad?
+            # Q is long long 8 bytes
+            # self.children = array('Q')
+
+        self.visited = 0
+        self.distance = 0
+        self.parent = None
+
+    def __setattr__(self, item, value):
+        self.__dict__[item] = value
```

Comment:
This is me just trying to be too concise in the [bfs](https://github.com/ice109/sublinear/blob/oo-graph-implementation/graph.py#L61).